examples/haskell/example.hs

```diff
@@ -0,0 +1,16 @@
+
+import OpenCog.AtomSpace.Api
+import OpenCog.AtomSpace.Types
+import Control.Monad.IO.Class
+import Data.Default
+
+main :: IO ()
+main = runOnNewAtomSpace program
+
+program :: AtomSpace ()
```

Comment:
It is rare, but there are times when multiple atomspaces need to be used at the same time.